src/test/regress/expected/multi_modifications.out

```diff
@@ -297,6 +297,16 @@ ERROR:  relation bidders is not distributed
 -- commands containing a CTE are supported
 WITH new_orders AS (INSERT INTO limit_orders VALUES (411, 'FLO', 12, '2017-07-02 16:32:15', 'buy', 66))
 DELETE FROM limit_orders WHERE id < 0;
+-- we have to be careful that modifying CTEs are part of the transaction and can thus roll back
+BEGIN;
```

Comment:
I think this test doesn't trigger the bug we've fixed since we're already inside `BEGIN`; So, `INSERT` is already sent inside a coordinated TX.

We should probably find a test where `DELETE` fails itself such as 
 (before the fix): 
```SQL
WITH orders AS (INSERT INTO users_table (user_id) VALUES (1)) DELETE FROM events_table WHERE user_id = 3 RETURNING user_id / 0;
DEBUG:  common table expressions are not supported in distributed modifications
DEBUG:  generating subplan 23_1 for CTE orders: INSERT INTO public.users_table (user_id) VALUES (1)
DEBUG:  Creating router plan
DEBUG:  Plan is router executable
DEBUG:  Plan 23 query after replacing subqueries and CTEs: DELETE FROM public.events_table WHERE (user_id OPERATOR(pg_catalog.=) 3) RETURNING (user_id OPERATOR(pg_catalog./) 0)
DEBUG:  Creating router plan
DEBUG:  Plan is router executable
LOG:  issuing INSERT INTO public.users_table_102009 (user_id) VALUES (1)
DETAIL:  on server localhost:9701
LOG:  issuing DELETE FROM public.events_table_102055 events_table WHERE (user_id OPERATOR(pg_catalog.=) 3) RETURNING (user_id OPERATOR(pg_catalog./) 0)
DETAIL:  on server localhost:9700
ERROR:  division by zero
CONTEXT:  while executing command on localhost:9700
```